### PR TITLE
Rename openid_wallet_relying_party to openid_credential_verifier

### LINKED ISSUE #10: rename openid_wallet_relying_party to openid_credential_verifier
since we have the name credential issuer, and therefore the enntity type identifier openid_credential_issuer, for sake of normalization and omogenity, it would be convenient to rename openid_wallet_relying_party to openid_credential_verifier

@selfissued WDYT?

_Originally posted by @peppelinux in https://github.com/peppelinux/federation-wallet/pull/9#discussion_r1720827029_

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #10 

Cc: @rohe @ve7jtb @vdzhuvinov

### CODE REVIEW

File: openid-federation-wallet-1_0.md [hunk 1]
```diff
@@ -267,11 +267,11 @@ For information on metadata parameters specific to OpenID Credential Issuers,
...
 the OpenID for Verifiable Credential Issuance [@!OpenID4VCI] specification.
 
-## OpenID Wallet Relying Party Entity Type
+## OpenID Credential Verifier Entity Type
 
-The OpenID Federation Entity Type Identifier for the Wallet Relying Party is `openid_wallet_relying_party`.
+The OpenID Federation Entity Type Identifier for the Credential Verifier is `openid_credential_verifier`.
 
-This specification introduces a distinct Entity Type Identifier for the OpenID Wallet Relying Party to clearly differentiate it from a traditional OpenID Connect Relying Party (`openid_relying_party`). This distinction highlights the unique characteristics and functionalities of the Wallet ecosystem and its Wallet Relying Party.
+This specification introduces a distinct Entity Type Identifier for the OpenID Credential Verifier to clearly differentiate it from a traditional OpenID Connect Relying Party (`openid_relying_party`). This distinction highlights the unique characteristics and functionalities of the Wallet ecosystem and its Credential Verifier.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:274]:
```suggestion
This specification introduces a distinct Entity Type Identifier for the OpenID Credential Verifier to clearly differentiate it from a traditional OpenID Connect Relying Party (`openid_credential_verifier`). This distinction highlights the unique characteristics and functionalities of the Wallet ecosystem and its Credential Verifier.
```

**Review by peppelinux — CHANGES_REQUESTED** [review]
only a quick fix on the metadata name, once you'll get it merged you can consider this PR approved and therefore merge it without any further revision by me

File: openid-federation-wallet-1_0.md (hunk 1 above)
Comment (selfissued) [reply openid-federation-wallet-1_0.md:274]: This suggestion is wrong.  We're actually talking about an OpenID Connect RP here.

**Review by peppelinux — APPROVED** [review]

**Review by rohe — APPROVED** [review]

**Review by ve7jtb — COMMENTED** [review]
lgtm

**Pull request merged**